### PR TITLE
1.0: update explanation about Elasticsearch and Kibana

### PR DESCRIPTION
**Pull request by kenhys**
Closes: #348

See https://www.elastic.co/pricing/faq/licensing#does-this-mean-that-elasticsearch-and-kibana-are-no-longer-open-source

### CODE REVIEW

File: container-deployment/docker-compose.md [hunk 1]
```diff
@@ -4,7 +4,10 @@ This article explains how to collect [Docker](https://www.docker.com/) logs and
 
 ![Kibana](../.gitbook/assets/7.10_kibana-homepage.png)
 
-[Elasticsearch](https://www.elastic.co/products/elasticsearch) is an open-source search engine known for its ease of use. [Kibana](https://www.elastic.co/products/kibana) is an open-source Web UI that makes Elasticsearch user-friendly for marketers, engineers and data scientists alike.
+[Elasticsearch](https://www.elastic.co/products/elasticsearch) was an open-source search engine known for its ease of use. [Kibana](https://www.elastic.co/products/kibana) was an open-source Web UI that makes Elasticsearch user-friendly for marketers, engineers and data scientists alike.
```
Comment (cosmo0920) [inline container-deployment/docker-compose.md]: was -> had been x2 

With small tense fixes, it should be fine.

Comment (kenhys) [reply container-deployment/docker-compose.md]: Thanks, I've fixed it.

**Review by cosmo0920 — APPROVED** [review]

**Comment by cosmo0920** [comment]
Let's :shipit:

**Comment by stockholmux** [comment]
thanks!

**Pull request merged**